docs/before/after-acceptance.md

```diff
@@ -9,4 +9,4 @@ You have received your admission offer letter? :sunglasses:
 
 <!-- If you are interviewing for a scholarship, click here to [know more](interview.md) -->
 
-Continue to our [after acceptance guide](../after/introduction.md)
\ No newline at end of file
+Continue to our [after acceptance guide](docs/after/introduction)
```

Comment:
Absolute paths yg lain are written as `/docs` instead of `docs`
Have you tested that this works? Just in case